### PR TITLE
Update Spack entries in cvmfsdirtab.sh

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
@smuzaffar ok to merge?

File: cvmfs/cvmfsdirtab.sh [hunk 1]
```diff
@@ -25,30 +25,28 @@ for cmsdir in "$@" ; do
   done
   if [ -d ${CVMFS_DIR}/$cmsdir/spack ]; then
     echo "/${cmsdir}/spack/share"
-    echo "/${cmsdir}/spack/share/data-l1trigger-l1tmuon"
-    echo "/${cmsdir}/spack/share/data-generatorinterface-evtgeninterface"
-    echo "/${cmsdir}/spack/share/data-magneticfield-interpolation/*"
+    echo "/${cmsdir}/spack/share/cms/data-l1trigger-l1tmuon"
+    echo "/${cmsdir}/spack/share/cms/data-generatorinterface-evtgeninterface"
+    echo "/${cmsdir}/spack/share/cms/data-magneticfield-interpolation/*"
```
Comment (smuzaffar) [inline cvmfs/cvmfsdirtab.sh:30]: what about cleanup of these extra `cms` paths?

Comment (iarspider) [reply cvmfs/cvmfsdirtab.sh:30]: I will take care of them in another PR, still checking.

**Comment by iarspider** [comment]
hold

I need to run the tests

**Comment by smuzaffar** [comment]
@iarspider , is this good to go in now?

**Comment by iarspider** [comment]
@smuzaffar not yet. If you are ok with changes in general, I will merge this myself once the tests are done.

**Comment by smuzaffar** [comment]
I am fine with current change

**Pull request merged**